.github/workflows/codeql-analysis.yml

```diff
@@ -33,7 +33,7 @@ jobs:
 
```

Comment:
Please update version.json to 2.12.30 so that I can create a new release after merging

Reply:
done